src/main/drivers/flash_m25p16.c

```diff
@@ -22,12 +22,15 @@
 
 #ifdef USE_FLASH_M25P16
 
-#include "flash.h"
-#include "flash_m25p16.h"
+#include "pg/flash.h"
```

Comment:
#includes should be in alphabetical order (I know many aren't, but we should be aiming for it).

Reply:
Agreed. Oversight on my part.